public/app/features/alerting/unified/Receivers.tsx

```diff
@@ -16,25 +19,58 @@ import { ReceiversAndTemplatesView } from './components/receivers/ReceiversAndTe
 import { useAlertManagerSourceName } from './hooks/useAlertManagerSourceName';
 import { useAlertManagersByPermission } from './hooks/useAlertManagerSources';
 import { useUnifiedAlertingSelector } from './hooks/useUnifiedAlertingSelector';
-import { fetchAlertManagerConfigAction, fetchGrafanaNotifiersAction } from './state/actions';
+import {
+  fetchAlertManagerConfigAction,
+  fetchContactPointsStateAction,
+  fetchGrafanaNotifiersAction,
+} from './state/actions';
+import { CONTACT_POINTS_STATE_INTERVAL_MS } from './utils/constants';
 import { GRAFANA_RULES_SOURCE_NAME } from './utils/datasource';
 import { initialAsyncRequestState } from './utils/redux';
 
+export interface NotificationErrorProps {
+  errorCount: number;
+}
+
+const NotificationError: FC<NotificationErrorProps> = ({ errorCount }: NotificationErrorProps) => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <div className={styles.warning}>
```

Comment:
Nice improvement! I like having everything error-related in a single component. 
You may also try the [Stack](https://github.com/grafana/grafana-experimental/blob/main/src/query-builder/Stack.tsx) component which helps to work with `flex` layouts.

Reply:
ohh!!! thank you @konrad147 !!! I will take a look! 👀

Reply:
Nice!